inventory/os.cf

```diff
@@ -14,4 +19,20 @@ _inventory_lsb_found::
   "description" string => "$(sys.flavor) (LSB missing)",
                   meta => { "inventory", "attribute_name=OS" };
 
+# Hard coded values for exceptions / platforms without os-release:
+
+centos_5::
+  "description" string => "CentOS Linux 5", # Matches format of os-release on 7+
```

Comment:
This extracts the major.minor but i guess PRETTY_NAME in os-release doens't have the minor version?
```

    centos::
      "description"
        string => concat( "CentOS ",
                          regex_replace( nth( string_split( nth( classesmatching( "centos_\d+_\d+" ), 0), "centos_", 2), 1), "_", ".", "")),
        meta => { "inventory", "attribute_name=OS" },
        if => not( isvariable( "sys.os_release" ) );

```

Reply:
Your solution is more general and attempting to address more cases, but also much more complicated, harder to read, and I think harder to understand from a user perspective.

I much prefer that "OS" in mission Portal, just matches what is in os-release as PRETTY_NAME. It is what the OS distro intended to be the "User facing" (pretty) name.

Like I've mentioned in the comment above, I chose the specific format "CentOS Linux 5" when os-release is missing, so it matches os-release in CentOS 7.

> i guess PRETTY_NAME in os-release doesn't have the minor version?

`PRETTY_NAME` on CentOS follows the format above, i.e. doesn't include minor version.